lib/onesie/template_reader.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+using Rainbow
+
+module Onesie
+  # Responsible for loading custom template files
+  class TemplateReader
+    def self.read_template(path)
+      unless File.exist?(path)
+        puts "Custom Template could not be found at #{path}".red
+        return
```

Comment:
@timlkelly this is to resolve the file being created despite the template not existing.

I was not able to find a way to prevent the generator to not create the file, even with raising an exception here, so I'm defaulting to an error message being printed, and in the consumer of this, defaulting back to the default template.